### PR TITLE
LP-196 Add hint links to LP name page

### PR DESCRIPTION
**Pull request by mattch1**
### JIRA link
https://companieshouse.atlassian.net/browse/LP-196
Sub task: https://companieshouse.atlassian.net/browse/LP-289

### Change description
This covers AC 2,3 and 4 only.

AC2: Added Companies House register link to the first paragraph.
AC3: added paragraph for sensitive word including the link.
AC4: Already complete, change the "opens in new tab" text to match the AC.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/name.njk [hunk 1]
```diff
@@ -56,13 +56,18 @@
...
+          <p>
+            If the name includes
+            <a href="https://www.gov.uk/government/publications/incorporation-and-names/annex-a-sensitive-words-and-expressions-or-words-that-could-imply-a-connection-with-government">sensitive words or expressions that require approval (opens in a new tab)</a
+            >. you must get permission before you register.
+          </p>
           <p>
             You should also use the
             <a href="https://www.gov.uk/search-for-trade mark" target="_blank"
-              >trade mark search (link opens in a new tab)</a
+              >trade mark search (opens in a new tab)</a
```
Comment (dlloyd1-ch) [inline src/views/name.njk:70]: The page in the Jira ticket is based on V5 of the prototype. Needs to use the 'insetText' nunjuck (https://github.com/companieshouse/limited-partnerships-prototype/blob/e10157ecbf2f52c1f6cc82bb3198c7693ea07c70/app/views/v5/registration/lp-name.html#L37)
Lines 49-77  can be replaced with this.

File: src/views/name.njk [hunk 2]
```diff
@@ -56,13 +56,18 @@
...
+            <a href="https://find-and-update.company-information.service.gov.uk/company-name-availability">Companies House register (opens in a new tab)</a
             >. If it is, we will reject your application.
           </p>
+          <p>
+            If the name includes
+            <a href="https://www.gov.uk/government/publications/incorporation-and-names/annex-a-sensitive-words-and-expressions-or-words-that-could-imply-a-connection-with-government">sensitive words or expressions that require approval (opens in a new tab)</a
+            >. you must get permission before you register.
+          </p>
           <p>
             You should also use the
```
Comment (sarahocarroll) [inline src/views/name.njk:68]: "https://www.gov.uk/search-for-trade mark" will give a page not found, needs to be "https://www.gov.uk/search-for-trademark"

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/name.njk (hunk 1 above)
Comment (mattch1) [reply src/views/name.njk]: Updated.

File: src/views/name.njk (hunk 2 above)
Comment (mattch1) [reply src/views/name.njk]: Updated.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/name.njk [hunk 3]
```diff
@@ -46,21 +46,13 @@
...
-            {{ i18n.namePage.checkNameExistingLP | safe }}
-          </p>
-          <p>
-            {{ i18n.namePage.checkNameExistingTrademark | safe }}
-          </p>
-        </div>
-      </details>
+      {{ govukInsetText({
+        html: 
+        '<p>You must check the name is not the same as an existing partnership or company on the <a href="https://find-and-update.company-information.service.gov.uk/company-name-availability">Companies House register (opens in a new tab)</a>. If it is, we’ll reject your application.</p>
```
Comment (markpit) [inline src/views/name.njk:51]: we should add the text to the translations.json files

Comment (mattch1) [reply src/views/name.njk]: Introduced localisation

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: locales/cy/translations.json [hunk 4]
```diff
@@ -11,7 +11,10 @@
...
         "checkNameExistingTrademark" : "WELSH - You should also use the <a href='https://www.gov.uk/search-for-trade mark' target='_blank'>trade mark search (link opens in a new tab)</a> to check if your name matches an existing trade mark. If it does, consider choosing a different name. Companies House will not carry out this check for you.",
         "nameEnding" : "WELSH - Which limited partnership name ending would you prefer to go on the public record?",
-        "nameEndingHint" : "WELSH - There's no difference between the name endings Limited Partnership, LP and L.P. except for how they look."
+        "nameEndingHint" : "WELSH - There's no difference between the name endings Limited Partnership, LP and L.P. except for how they look.",
+        "existingPartnershipHint" : "You must check the name is not the same as an existing partnership or company on the <a href='https://find-and-update.company-information.service.gov.uk/company-name-availability'>Companies House register (opens in a new tab)</a>. If it is, we’ll reject your application.",
```
Comment (markpit) [inline locales/cy/translations.json:15]: can you start the welsh text with "WELSH - " , it will help us to identify the welsh text when testing.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by markpit — APPROVED** [review]

File: locales/cy/translations.json (hunk 4 above)
Comment (mattch1) [reply locales/cy/translations.json]: These sentences are now in Welsh.

**Review by sarahocarroll — APPROVED** [review]

**Pull request merged**